framework/include/utils/MooseTypes.h

```diff
@@ -161,8 +163,8 @@ enum VarKindType
 
 enum KernelType
 {
-  KT_TIME = 0,
-  KT_NONTIME = 1,
+  KT_NONTIME = 0,
```

Comment:
just curious, why is this change?

Reply:
We want tag id for KT_NONTIME be zero because it is default tag.